apps/web/components/Organization/Slice/Districts/DistrictsSlice.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Districts, Organization } from '@island.is/web/graphql/schema'
+import {
+  Box,
+  Button,
+  GridColumn,
+  GridContainer,
+  GridRow,
+} from '@island.is/island-ui/core'
+import * as styles from './DistrictsSlice.treat'
+import Link from 'next/link'
+
+interface SliceProps {
+  slice: Districts
+  organization?: Organization
+}
+
+export const DistrictsSlice: React.FC<SliceProps> = ({
+  organization,
+  slice,
+}) => {
+  return (
+    <>
+      <section key={slice.id} aria-labelledby={'sliceTitle-' + slice.id}>
+        <GridContainer>
+          <Box
+            borderTopWidth="standard"
+            borderColor="standard"
+            paddingTop={[8, 6, 15]}
+            paddingBottom={[4, 5, 10]}
+          >
+            <h2 id={'sliceTitle-' + slice.id} className={styles.districtsTitle}>
+              {slice.title}
+            </h2>
+            <GridRow>
+              <GridColumn span={['12/12', '12/12', '7/12']}>
+                <ul className={styles.districtsList}>
+                  {slice.links.map((link) => (
+                    <li className={styles.districtsListItem}>
+                      <Link href={link.url}>
+                        <a>
+                          <Button variant="text">{link.text}</Button>
+                        </a>
+                      </Link>
```

Comment:
Just wondering, do we need the anchor here?
```suggestion
                      <Link href={link.url}>
                        <Button variant="text">{link.text}</Button>
                      </Link>
```

Reply:
We need the anchor so that the browser recognizes them as links (the `Link` component automatically passes the href to it's children). Although this is probably not the best method. 

There seem to be quite a few non-anchor links on island.is, shouldn't that be fixed? For example they can't be right clicked and opened in a new tab.

Reply:
`Link` renders an anchor for you, whether it's wrapped in Next link or not so this anchor is unnecessary, it means that it will be rendered like so:
```
<a>
 <a>
  <Button></Button>
 </a>
</a>
```

Reply:
The problem was that we were importing `Link` from `next/link`, that did not render anchors.